Scheduler/feature_scheduler/maintel/fbs_config_fieldsurvey.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of ts_config_ocs.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from lsst.ts.fbs.utils.data.field_survey_centers import get_sv_fields
+from lsst.ts.fbs.utils.maintel.basis_functions import get_basis_functions_field_survey
+from lsst.ts.fbs.utils.maintel.detailers import get_detailers_field_survey
+from lsst.ts.fbs.utils.maintel.make_fieldsurvey_scheduler import (
+    MakeFieldSurveyScheduler,
+)
+
+
+def get_scheduler():
+    """Construct feature based scheduler.
+
+    Returns
+    -------
+    nside : int
+        Healpix map resolution.
+    scheduler : Core_scheduler
+        Feature based scheduler.
+    """
+
+    nside = 32
+
+    make_scheduler = MakeFieldSurveyScheduler(nside=nside, ntiers=1)
+
+    nvisits = {"u": 20, "g": 20, "r": 20, "i": 20, "z": 20, "y": 20}
+    sequence = "ugrizy"
+    # exposure time in seconds
+    exptimes = {"u": 38, "g": 30, "r": 30, "i": 30, "z": 30, "y": 30}
+    # 1 --> single 30 second exposure
+    nexps = {"u": 1, "g": 1, "r": 1, "i": 1, "z": 1, "y": 1}
+
+    wind_speed_maximum = 13.0  # maximum direct wind in m/s
+
+    field_survey_kwargs = {
+        "nvisits": nvisits,
+        "sequence": sequence,
+        "exptimes": exptimes,
+        "nexps": nexps,
+    }
+
+    # This might move to be part of a separate function
+    basis_functions = get_basis_functions_field_survey(
+        nside=nside,
+        wind_speed_maximum=wind_speed_maximum,
+    )
+
+    detailers = get_detailers_field_survey()
+
+    # Specify the json BLOCK to be used
+    json_block = "MAINTEL_COMCAM_IMAGING"
+    observation_reason = "science"
```

Comment:
Conceptually I have a question for why observation_reason is set separately in FieldSurvey and in the .json block.

Reply:
Hmm, that's a good question. I don't actually know what happens to the observation_reason that gets set in the FieldSurvey. I only know that it is not passed by the scheduler to the json block. @rhiannonlynne?

Reply:
Well, this is a feature based scheduler metadata. We don't pass this information to the scripts though, but I guess we could.

Reply:
We should pass it to the scripts. 
We should probably set metadata at the earlier opportunity, so if the FBS sets 
`observation_reason`, `science_program`, `target_name` 
those should be passed to the JSON BLOCK and then into the headers. 
(likewise, `scheduler_note` needs to be passed into the headers so it can get into the ConsDB, which I think means it needs to go through the JSON BLOCK too)

Reply:
But maybe Keith needs to explain why it's set separately?

Reply:
I agree, passing it through from the FBS to the script config would be great, that way we have one source of truth. I think right now the only reason it is set separately because we are missing that pass-through connection. Is that something we could add quickly @tribeiro or should we just merge this as-is and then patch it later when we have the pass-through setup?

Reply:
I asked because I was surprised that the observation_reason is set separately. I had expected that observation_reason would be passed to the FieldSurvey and then would go through the JSON block

Reply:
As with many things, there are many ways to get this set .. but yes it could be set by the user before it gets to FieldSurvey (it is a keyword argument for FieldSurvey, which gets passed to the detailer which sets these per Target). 

The part I don’t know is it Tiago has wired up the output of observation_reason in the requested observation returned by the FBS into the JSON BLOCK.

Reply:
Think this needs to be addressed in a separate ticket.